app/components/Nav.tsx

```diff
@@ -80,14 +80,13 @@ export default function Nav() {
 							className="nav-link d-inline-block px-2"
 						>
 							<svg
-								role="img"
-								viewBox="0 0 24 24"
-								height="24"
-								width="24"
 								xmlns="http://www.w3.org/2000/svg"
+								width="24"
+								height="24"
+								viewBox="0 0 512 512"
```

Comment:
Hi @antriksh-9,
Is there any reason to remove the `role="img"` attribute and change the `viewBox` dimension?